### DISCUSSION TITLE
Images don't load, can't use the media library

### DISCUSSION THREAD

**Discussion by TejasKaranji** [mitchmac/ServerlessWP#47] [Q&A]
I'm using serverless wp deployed on vercel+planetscale, and I'm unable to upload any images to my site. The upload appears to happen successfully, but I cannot access the uploaded images. Please help?

**Comment by mitchmac** [comment]
Hi, serverless functions don't store files for a guaranteed duration so uploaded files need to be stored somewhere like S3.

There's a quick note in the project's readme that may help you set up the required WP Offload Media Lite for Amazon S3 plugin.

**Reply by lvillalobos-cr** [reply]
Hmm I followed the guide and created the S3 keys via the plugins documentation, but now when I go to click "save and continue" on the plugin screen in the settings menu nothing happens. Is there anymore documentation on this?

**Reply by TejasKaranji** [reply]
Images in the UI don't load as well.. I installed elementor, and none of the logos and icons of the plugin are able to load. Is this unique to me or expected behavior?

**Comment by lutfi-haslab** [comment]
can i use compatible S3 like R2 cloudflare or Backblaze?
